references/cli/delete.go

```diff
@@ -75,21 +75,29 @@ func NewDeleteCommand(c common2.Args, order string, ioStreams cmdutil.IOStreams)
 		if err != nil {
 			return err
 		}
-		o.ForceDelete = force
+		yes, err := cmd.Flags().GetBool("yes")
+		if err != nil {
+			return err
+		}
+		o.ForceDelete = force || yes
```

Comment:
you need to add a new flag to represent `yes`. `--force` and `--yes` are not the same.